android/data/src/main/java/org/fossasia/openevent/data/Feedback.java

```diff
@@ -0,0 +1,37 @@
+package org.fossasia.openevent.data;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.github.jasminb.jsonapi.LongIdHandler;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+import org.fossasia.openevent.core.auth.model.User;
+
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Data
+@Builder
+@Type("feedback")
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString(exclude = "event")
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
+@EqualsAndHashCode()
```

Comment:
Why no callSuper = false?

Reply:
i thought callSuper = false is by default, will make the changes